projects/ngx-charts-on-fhir/src/lib/fhir-chart-summary/summary.service.spec.ts

```diff
@@ -29,19 +37,19 @@ describe('ScatterDataPointSummaryService', () => {
     it('should include statistics for current and previous timespan', () => {
       const statisticsService = jasmine.createSpyObj<StatisticsService>('StatisticsService', ['getFormattedStatistics']);
       const summaryService = new ScatterDataPointSummaryService(statisticsService);
-      statisticsService.getFormattedStatistics.and.callFake((layer, { min, max }) => ({
+      statisticsService.getFormattedStatistics.and.callFake((mockLayer, { min, max }) => ({
         min: String(min),
         max: String(max),
       }));
```

Comment:
why did you change this (unused) parameter name? it's now shadowing the mockLayer const from module scope, which could be confusing.